src/WebUI/locales/en.yml

```diff
@@ -971,6 +972,10 @@ user:
         navigate: Navigate to our {modMailLink} channel
         follow: Follow the instructions in that channel to reach out to us
       outro: Please note that at this time, we do not accept appeals through any platform other than Discord.
+  notifications:
+    title: Notifications
+    empty: you have no notifications yet
+    toAll: All Notifications
```

Comment:
these aren't being used - also we need rus / cn

Reply:
removed the unused translation
Ru and cn will not be done yet